src/calz_unred.jl

```diff
@@ -52,16 +52,16 @@ microns).
 Estimate how a flat galaxy spectrum (in wavelength) between \$1200 Å\$ and
 \$3200 Å\$ is altered by a reddening of E(B-V) = 0.1.
 
-``` julia
+```jldoctest
```

Comment:
You can either leave `julia` here or switch to `jldoctest` but by making it a real test, for example in this way:
````
```jldoctest
julia> wave = collect(1200:50:3150);

julia> flux = ones(wave);

julia> flux_new = calz_unred.(wave, flux, -0.1)
40-element Array{Float64,1}:
 0.32751 
 0.340493
 0.351806
 0.361775
[...]
```
````
The `[...]` can be used to truncate the output, that otherwise would be too long.